### PR TITLE
Feat/use interval

### PR DESCRIPTION
**Pull request by calilrenner**

### CODE REVIEW

File: src/pages/Timeline.js [hunk 1]
```diff
@@ -18,7 +18,30 @@ import {
...
+      if(delay !== null) {
+        let update = setInterval(newQueuePosts, delay);
+        return () => clearInterval(update);
+      }
+    }, [delay])
+  }
+
+  useInterval(() => {
+    setOnChangePost(!onChangePost)
+    console.log(onChangePost)
```
Comment (leoguzi) [inline src/pages/Timeline.js:43]: console.log left behind!

Comment (leoguzi) [inline src/pages/Timeline.js]: console.log left behind!

**Review by leoguzi — CHANGES_REQUESTED** [review]

File: src/pages/Timeline.js (hunk 1 above)
Comment (calilrenner) [reply src/pages/Timeline.js]: console removed

**Review by leoguzi — APPROVED** [review]
Nice job! Everything looks fine.

**Comment by sidneyouteiro** [comment]
Just fix the merge conflicts, then i will approve 👍🏽

**Comment by calilrenner** [comment]
Fixed conflicts.

**Review by sidneyouteiro — APPROVED** [review]

**Pull request merged**